tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -103,7 +107,7 @@ def allow_dirty(pytestconfig):
 
 @pytest.fixture(scope="module")
 def enable_auth(pytestconfig):
-    return pytestconfig.getoption("enable_auth")
+    return True if pytestconfig.getoption("enable_auth").lower() == "true" else False
```

Comment:
I think we need at least a single test that fails if it expects a failure due to authentication being enabled. So basically it should send no auth details and get an UNAUTHENTICATED response. That way we know that something is wrong.